setup.py

```diff
@@ -79,35 +49,34 @@ def build_extension(self, ext, cmake):
         cfg = 'Debug' if self.debug else 'Release'
         build_args = ['--config', cfg]
 
-        if platform.system() == "Windows":
-            cmake_args += ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}'.format(
-                cfg.upper(),
-                extdir)]
-            cmake_args += ['-G', os.getenv('MORPHIO_CMAKE_GENERATOR',
-                                           "Visual Studio 15 2017 Win64")]
-        else:
-            cmake_args += ['-DCMAKE_BUILD_TYPE={}'.format(cfg),
-                           '-DMorphIO_CXX_WARNINGS=OFF',
-                           ]
-            build_args += ["--", "-j{}".format(max(MIN_CPU_CORES, get_cpu_count())),
-                           ]
+        cmake_args += ['-DCMAKE_BUILD_TYPE={}'.format(cfg),
+                       '-DMorphIO_CXX_WARNINGS=OFF',
+                       '-GNinja',
```

Comment:
Neat (note to self: requires new dependencies in Spack)

Reply:
yeah, that's why I added you to the review; it should be ok though, right? adding `ninja`?

Reply:
`depends_on('ninja', type='build')` should probably be good enough.